app/frontend/src/features/communities/events/EventPage.tsx

```diff
@@ -218,7 +230,7 @@ export default function EventPage() {
                     >
                       {VIRTUAL_EVENT}
                     </Typography>
-                    <Link href={event.onlineInformation.link}>
+                    <Link href={event.onlineInformation.link} variant="body1">
```

Comment:
Does this make a difference?

Reply:
I swear it did before, not anymore probably because of the theme changes from the donation PR... removed.